src/funding/funding.html

```diff
@@ -40,62 +35,78 @@ <h1 class="title heading heading1 gradient">
                 given period of time. Please go to the Deposits below and withdraw your funds. You
                 can withdraw your funds at any time. Funds are safe.
               </span>
-              <span if.to-view="!fundingFailed && !deal.isTargetReached">
-                You're depositing on behalf of ${deal.daoRepresentedByCurrentAccount.name}. This deal needs to
-                be funded and executed within ${fundingDaysLeft} days.
+              <span
+                if.to-view="!fundingFailed && !deal.isTargetReached && !deal.isUserProposalLead"
+              >
+                You're depositing on behalf of ${deal.firstDao.name}. This deal needs to be funded
+                and executed within ${fundingDaysLeft} days.
+              </span>
+              <span if.to-view="!fundingFailed && !deal.isTargetReached && deal.isUserProposalLead">
+                This deal needs to be funded and executed within ${fundingDaysLeft} days.
               </span>
               <span if.to-view="!fundingFailed && deal.isTargetReached">
-                All parties have deposited their funds. Anyone can now execute the swap.
+                All parties have deposited their funds. Anyone can now execute the swap on the deal
+                dashboard.
               </span>
             </p>
-            <div class="links">
-              <address-link
-                address.to-view="deal.daoDepositContracts.get(deal.daoRepresentedByCurrentAccount).address"
-                link-text="DAODepositManager Contract"
-                show-arrow-inside-link.bind="true"
-                show-copy-icon.bind="false">
-              </address-link>
-              <address-link
-                address.to-view="deal.moduleContract.address"
-                link-text="TokenSwapModule Contract"
-                show-arrow-inside-link.bind="true"
-                show-copy-icon.bind="false">
-              </address-link>
-            </div>
           </div>
-          <div class="deposit">
-            <div
-              class="form"
-              if.to-view="!deal.isUserProposalLead && !deal.isTargetReached && !fundingFailed">
+          <div
+            class="deposit"
+            if.to-view="!deal.isUserProposalLead && !deal.isTargetReached && !fundingFailed"
```

Comment:
Nit: Can we have separate vars to keep track of the state?
Eg. in line 39, we have the same check

Reply:
I added a new var for that state for you in a <let> tag.